.circleci/config.yml

```diff
@@ -447,80 +423,74 @@ jobs:
           template: basic_fail_1
 
   deploy_dev:
-    docker:
-      - image: cimg/ruby:3.2.2
+    executor:
+      name: tariff/ruby
     environment:
-      SENTRY_ENVIRONMENT: "development"
+      SENTRY_ENVIRONMENT: development
     steps:
       - queue/until_front_of_line:
           time: "10"
           consider-branch: false
           dont-quit: true
       - cf_deploy_docker:
-          docker_image_tag: dev-$CIRCLE_SHA1
-          space: "development"
-          environment_key: "dev"
-          app_domain_prefix: "dev"
-      - sentry-release
+          docker_tag: dev-$CIRCLE_SHA1
+          space: development
+          environment_key: dev
+          app_domain_prefix: dev
+      - tariff/sentry-release:
+          environment: development
 
   deploy_review:
-    docker:
-      - image: cimg/ruby:3.2.2
+    executor:
+      name: tariff/ruby
     environment:
-      SENTRY_ENVIRONMENT: "development"
+      SENTRY_ENVIRONMENT: development
     steps:
       - queue/until_front_of_line:
           time: "10"
-          consider-branch: true
+          consider-branch: false
           dont-quit: true
       - cf_deploy_review
 
-  purge_reviews:
-    docker:
-      - image: cimg/ruby:3.2.2
-    environment:
-      SENTRY_ENVIRONMENT: "development"
-    steps:
-      - cf_purge_reviews
-
   deploy_main_to_staging:
-    docker:
-      - image: cimg/ruby:3.2.2-node
+    executor:
+      name: tariff/ruby
```

Comment:
Should this be a ruby-node executor?

Reply:
So this uses the orb `ruby` executor, which [defaults](https://github.com/trade-tariff/trade-tariff-ci-orb/blob/main/src/executors/ruby.yml) to `3.2.2-node`.

Reply:
Oh. I guess the inverse of this is that when we only want a smaller ruby image we end up with node as well?